### DISCUSSION TITLE
After update Docker : 500: Internal Error

### DISCUSSION THREAD

**Discussion by Khampol** [open-webui/open-webui#7171] [General]
Hi,
 I use open-webui for a while, so far so good, I update it, I use Docker, and use this method 

https://docs.openwebui.com/getting-started/quick-start/#option-2-manual-update

After that I try reboot the VM then get "500: Internal Error".

If I process again to update docker it will work, but I f I reboot the VM, after reboot I get the 500: Internal Error

Look something wrong with docker after restart, docker do not boot or something.... ??...
Need help.

Update : I have to re-start it manually ( docker restart $(docker ps -a -q) ) to be able to use it after reboot VM.
You guys do not have this problem ? :-/

**Comment by Khampol** [comment]
ok find by myself : after update and reboot, the container stop (why?...). So after install/update new container need this : 

`docker update --restart=always <container ID>`

check ID by 

`docker ps`

Et voila.

**Reply by skhaz** [reply]
I have the same issue, thank you!
